lib/proxy/log_buffer/decorator.rb

```diff
@@ -10,17 +10,15 @@ def initialize(logger, buffer = Proxy::LogBuffer::Buffer.instance)
 
     def add(severity, message = nil, progname = nil, backtrace = nil, a_module = 'core', &block)
```

Comment:
Heh, you should leave a comment so I would not bother reading about how to contribute to Ruby :-)

https://bugs.ruby-lang.org/issues/12054

It's been fixed in trunk already. Interesting article, the speedup was not so obvious. I will remove this than.